somnio_starter/__brick__/{{project_name.snakeCase()}}/lib/app/router/auth_stream_scope.dart

```diff
@@ -0,0 +1,41 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+import 'package:{{project_name}}/app/app.dart';
+
+class AuthStreamScope extends InheritedNotifier<AuthStream> {
+  AuthStreamScope({
+    required AppBloc appBloc,
+    required super.child,
+    super.key,
+  }) : super(notifier: AuthStream(appBloc, appBloc.stream));
+
+  static AuthStream of(BuildContext ctx) =>
+      ctx.dependOnInheritedWidgetOfExactType<AuthStreamScope>()!.notifier!;
+}
+
+class AuthStream extends ChangeNotifier {
+  AuthStream(
+    AppBloc bloc,
+    Stream<dynamic> stream,
+  ) : _bloc = bloc {
+    notifyListeners();
```

Comment:
Is this needed?

Reply:
I tested the feature without this line and it seems to be working fine. 
I guess the purpose was to notify the status before building the widget but it seems to be left over